lib/pact_broker/matrix/resolved_selector.rb

```diff
@@ -211,9 +211,9 @@ def description
           "a version of #{pacticipant_name} with tag #{tag} (no such version exists)"
         elsif environment_name && pacticipant_version_number
           prefix = one_of_many? ? "one of the versions" : "the version"
-          "#{prefix} of #{pacticipant_name} currently deployed or released to #{environment_name} (#{pacticipant_version_number})"
+          "#{prefix} of #{pacticipant_name} currently in #{environment_name} (#{pacticipant_version_number})"
```

Comment:
FFS, this was on master but not pushed when I created the branch. I can't work out how to make github realise it's not part of the PR.